tests/generation/test_utils.py

```diff
@@ -1851,75 +1851,6 @@ def test_generate_continue_from_past_key_values(self):
                         )
                     )
 
-    @parameterized.expand([(1, False), (1, True), (4, False)])
-    @pytest.mark.generate
-    def test_new_cache_format(self, num_beams, do_sample):
```

Comment:
This test no longer makes sense: many models no longer accept or return the legacy cache format. This test was in practice checking whether generating with a `Cache` object is the same as generating... with a `Cache` object :D

We still have integration tests to ensure the two cache formats are equivalent, e.g. [this one](https://github.com/huggingface/transformers/blob/c23a1c19322152dbc89cfaa5f22310bd0afb92d0/tests/utils/test_cache_utils.py#L53).